installers/ansible/inventory.yaml

```diff
@@ -0,0 +1,31 @@
+---
+    all:
+        hosts:
+            localhost:
+        vars:
+            ansible_connection: local
+            ansible_python_interpreter: "/usr/bin/env python"
```

Comment:
Just curious, does the deployer container require `ansible_python_interpreter`?  If not, wondering if this is a good chance to remove this, and just let it auto-detect (or just explicitly define for the deployer if it needs it).  I know I always have to delete this setting and just let it auto-detect python 3 on my machine.

Anyway, not a big deal right now either way (not a blocker or anything), just figured I'd ask while we're touching this part of the `inventory`.

Reply:
I can test that out and see

Reply:
Per our discussions this is OBE at this time, and we can therefore defer looking into this.  I will mark this conversation as resolved.